src/app/containers/list/list.component.html

```diff
@@ -38,7 +38,7 @@
               (tool?.toolname ? '/' + tool?.toolname : '') }}</a>
           </td>
           <td>
-            {{ tool?.starredUsers.length === 0 ? '' : tool?.starredUsers.length }}
```

Comment:
If I'm reading this right, I think I still prefer blank over 0 stars.
Thoughts?